core/conversion/converters/BUILD

```diff
@@ -84,6 +84,7 @@ cc_library(
         "impl/topk.cpp",
         "impl/unary.cpp",
         "impl/unsqueeze.cpp",
+        "impl/index_put.cpp"
```

Comment:
Make sure to add this file to the `CMakeLists.txt` as well

Reply:
added it in CMakeLists